### PR TITLE
Update README.md file.

### PR DESCRIPTION
**Pull request by p-lenart**
Hello 

Today, I tried to generate FishyJoes bindings for the Tesseract library that we work on. While working with a README file, I found a few problems that can be addressed or clarified.

1) Is there any assumption about a local directory structure? Do we need to have all repositories like `FishyJoes`, `CriGeo`, and `CriGeo-bindings` at the same directory? I can see that generated `Package.swift` is using `../` when accessing packages.

2) Why do we need "Rancher Desktop"? I can't see any explanation in the README about the tools that we need to install.

3) Is it OK to modify generated `Package.swift`? I had problems with `platforms: [.macOS(.v11)],` mismatch between Tesseract and Tesseract-bindings.

4) I can't access "[Tutorial by Matt](https://cricut-my.sharepoint.com/personal/mstoker_cricut_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fmstoker%5Fcricut%5Fcom%2FDocuments%2FRecordings%2FClient%20Enablement%20Demos%2D20220505%5F145944%2DMeeting%20Recording%2Emp4&parent=%2Fpersonal%2Fmstoker%5Fcricut%5Fcom%2FDocuments%2FRecordings&ga=1)"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,14 +67,15 @@ sudo installer -pkg swift-wasm.pkg -target /
 
 Install Docker:
  a. `brew install rancher`
- b. open "Rancher Desktop.app"
- c. click "accept"
- d. enter password to finish install
- e. Authenticate with your github PAT: `nerdctl login ghcr.io`
+ b. create missing folder on macOS: `sudo mkdir /private/etc/sudoers.d`
```
Comment (p-lenart) [inline README.md]: Without it, I couldn't get "Rancher Desktop" working at all and the error message is very unreadable.

File: README.md [hunk 2]
```diff
@@ -67,14 +67,15 @@ sudo installer -pkg swift-wasm.pkg -target /
...
+ c. open "Rancher Desktop.app"
+ d. click "accept"
+ e. enter password to finish install
+ f. authenticate with your github PAT: `nerdctl login ghcr.io`
 
 1. In a new git repository, named `YourAwesomeLibrary-bindings`:
 ```
-mint run --executable fishy-joes cricut/FishyJoes package-init
+GITHUB_USER=... GITHUB_TOKEN=... mint run --executable fishy-joes cricut/FishyJoes package-init
```
Comment (p-lenart) [inline README.md:78]: Without providing `GITHUB_USER` and `GITHUB_TOKEN` we can't finish the `package-init` script. There is an error while running gradle, and it's a bit misleading.

**Comment by cobbal** [comment]
Thanks for the contribution!

1. The "../" assumptions are useful for local development and faster iteration before sending everything to the CI (there they should be commented out and the proper URLs used)
2. The current android toolchain runs from a docker image. If you want to compile locally and test on an android device/emulator, you will need some sort of docker runner. If you don't need this, it can be skipped.
3. Yes, modifying Package.swift is frequently necessary. The template is meant just as a starting point here.
4. I'll ask

File: README.md (hunk 1 above)
Comment (cobbal) [reply README.md]:
```suggestion
 b. create missing folder on macOS: `sudo mkdir -p /private/etc/sudoers.d`
```

The "-p" flag will let the command succeed if the folder already exists

**Review by cobbal — APPROVED** [review]

**Pull request merged**